### PR TITLE
i18n(it): tutorial setup - page 4 and 5

### PR DESCRIPTION
**Pull request by csarnataro**
#### What kind of changes does this PR include?

- Translated content

#### Description
    Added Italian translations for page 4 and 5 of the setup section of the tutorial

#### Related Issue / Implementation PR

- Closes Issue #

### CODE REVIEW

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Esistono modi diversi per salvare su GitHub il codice che hai in locale, tuttavia in questo tutorial useremo un metodo che non richiede di usare git dalla linea di comando. 
+
+1. Accedi con le tue credenziali a GitHub.com con il browser e fai click sul <kbd>+</kbd> nell'angolo in alto a destra dello schermo per creare un nuovo repository. 
+
+2. Scegli un nome per il tuo repository. Non deve necessariamente coincidere col nome della cartella del progetto.
+
+3. Ti verranno mostrate alcune opzioni, ma non è necessario modificare i valori di default. Scorri in basso e fai clic sul pulsante <kbd>Create Repository</kbd>.
+
+4. Ti verranno presentate altre opzioni per i passi successivi, ma non ne avrai bisogno, semplicemente prendi nota dell'URL del tuo repository. A questo  punto puoi uscire dalla pagina senza fare nient'altro.
```
Comment (lutaok) [inline src/content/docs/it/tutorial/1-setup/4.mdx:36]: Whitespace removal
```suggestion
4. Ti verranno presentate altre opzioni per i passi successivi, ma non ne avrai bisogno, semplicemente prendi nota dell'URL del tuo repository. A questo punto puoi uscire dalla pagina senza fare nient'altro.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Salvare (commit) il codice locale su GitHub
+
+Nella sezione precedente hai fatto alcune modifiche al contenuto della pagina. Questo significa che i file del tuo progetto sono cambiati, quindi VS Code mosterà un numero sopa l'icona "Source" nel menu. La scheda "Source" ti sarà utile per fare il commit dei tuoi file periodicamente su GitHub.
+
+1. Fai clic sulla scheda "Source Control" in VS Code per vedere un elenco di file che sono cambiati. Se vedi un messaggio che ti chiede di installare `git`, segui le istruzioni e poi riavvia VS Code.
+
+2. Fai clic sul menu <kbd>•••</kbd> "3 puntini" appena sopra il messaggio di commit e scegli <kbd>Remote</kbd> > <kbd>Add Remote</kbd>.
+
+3. Selezione <kbd>Add remote from GitHub</kbd>. Se necessario autenticati seguento le istruzioni, poi ritorna a VS Code e ripeti questa azione.
```
Comment (lutaok) [inline src/content/docs/it/tutorial/1-setup/4.mdx:47]: There are some typos:

```suggestion
3. Seleziona <kbd>Add remote from GitHub</kbd>. Se necessario autenticati seguendo le istruzioni, poi ritorna a VS Code e ripeti questa azione.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+5. Nella parte superiore del pannello è possibile inserire un **messaggio di commit** (cioè la descrizione dei cambiamenti ai tuoi file). Scrivi `commit iniziale` e fai clic sul pulsante <kbd>Commit</kbd> per fare il commit delle tue modifiche.
+
+6. Potresti vedere un messaggio che ti informa che non hai "staged commits", e che ti chiede se vuoi inserirli nell'area di stage. Fai clic su <kbd>Always</kbd> e continua.
+
+7. Infine, l'elenco dei file modificato dovrebbe essere sostituito da un pulsante <kbd>Publish</kbd>. Fai clic per inviare i tuoi file salavati su GitHub.
+
+### Visualizza il tuo progetto su GitHub
+
+Per verificare se il tuo progetto è stato salvato su GitHub, visita GitHub.com e cerca all'interno del tuo account l'elenco dei tuoi repository. Scegli l'ultimo che hai creato e verifica che contiene i file del tuo progetto Astro.
```
Comment (lutaok) [inline src/content/docs/it/tutorial/1-setup/4.mdx:59]:
```suggestion
Per verificare se il tuo progetto è stato salvato su GitHub, visita GitHub.com e cerca all'interno del tuo account l'elenco dei tuoi repository. Scegli l'ultimo che hai creato e verifica che contenga i file del tuo progetto Astro.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+## Checklist
+
+<Checklist>
+- [ ] Ho salvato il mio progetto su GitHub.
+</Checklist>
+</Box>
+
+### Resources
+
+- <p>[Utilizzare il sistema di controllo delle versioni Git in VS Code](https://code.visualstudio.com/docs/sourcecontrol/overview#_git-support) <Badge>external</Badge></p>
```
Comment (lutaok) [inline src/content/docs/it/tutorial/1-setup/4.mdx:71]:
```suggestion
- <p>[Utilizzare il sistema di versionamento Git in VS Code](https://code.visualstudio.com/docs/sourcecontrol/overview#_git-support) <Badge>external</Badge></p>
```

File: src/content/docs/it/tutorial/1-setup/5.mdx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Aggiungi il tuo repository GitHub come nuova appliazione Netlify
```
Comment (lutaok) [inline src/content/docs/it/tutorial/1-setup/5.mdx:19]: Typo here
```suggestion
  - Aggiungi il tuo repository GitHub come nuova applicazione Netlify
```

File: src/content/docs/it/tutorial/1-setup/5.mdx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+:::
+
+## Crea un nuovo sito Netlify
+
+1. Crea un account gratuito su [Netlify](https://netlify.com) se ancora non ne hai uno. 
+    Prendi nota del tuo username. Potrai vedere il tuo pannello di controllo a tutti i siti che hai creato all'indirizzo `https://app.netlify.com/teams/username`
+
+2. Fai clic su <kbd>Add new site</kbd> > <kbd>Import an existing project</kbd>. 
+
+    Ti verrà richiesto di connetterti a un servizio Git. Scegli GitHub e segui i passi che ti vengono mostrati a video per autenticare il tuo accoung GitHub.
```
Comment (lutaok) [inline src/content/docs/it/tutorial/1-setup/5.mdx:36]: Typo here:

```suggestion
    Ti verrà richiesto di connetterti a un servizio Git. Scegli GitHub e segui i passi che ti vengono mostrati a video per autenticare il tuo account GitHub.
```

**Review by lutaok — CHANGES_REQUESTED** [review]

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+title: Salva il tuo codice in un repository online
+description: |-
+  Tutorial: Costruisci il tuo primo blog Astro —
+  Crea un repository GitHub per il progetto del tutorial
+i18nReady: true
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:7]:
```suggestion

```

This is just for the primary version... it's not really clear from the variable itself.

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+2. Scegli un nome per il tuo repository. Non deve necessariamente coincidere col nome della cartella del progetto.
+
+3. Ti verranno mostrate alcune opzioni, ma non è necessario modificare i valori di default. Scorri in basso e fai clic sul pulsante <kbd>Create Repository</kbd>.
+
+4. Ti verranno presentate altre opzioni per i passi successivi, ma non ne avrai bisogno, semplicemente prendi nota dell'URL del tuo repository. A questo  punto puoi uscire dalla pagina senza fare nient'altro.
+
+
+## Salvare (commit) il codice locale su GitHub
+
+Nella sezione precedente hai fatto alcune modifiche al contenuto della pagina. Questo significa che i file del tuo progetto sono cambiati, quindi VS Code mosterà un numero sopa l'icona "Source" nel menu. La scheda "Source" ti sarà utile per fare il commit dei tuoi file periodicamente su GitHub.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:41]:
```suggestion
Nella sezione precedente hai fatto alcune modifiche al contenuto della pagina. Questo significa che i file del tuo progetto sono cambiati, quindi VS Code mosterà un numero sopa l'icona "Sorgente" nel menu. La scheda "Sorgente" ti sarà utile per fare il commit dei tuoi file periodicamente su GitHub.
```

VS Code is available in Italian, so we translate the menu names.

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Fai clic sulla scheda "Source Control" in VS Code per vedere un elenco di file che sono cambiati. Se vedi un messaggio che ti chiede di installare `git`, segui le istruzioni e poi riavvia VS Code.
+
+2. Fai clic sul menu <kbd>•••</kbd> "3 puntini" appena sopra il messaggio di commit e scegli <kbd>Remote</kbd> > <kbd>Add Remote</kbd>.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:45]:
```suggestion
2. Fai clic sul menu <kbd>•••</kbd> "3 puntini" appena sopra il messaggio di commit e scegli <kbd>Repository remoto</kbd> > <kbd>Aggiungi repository remoto</kbd>.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+3. Ti verranno mostrate alcune opzioni, ma non è necessario modificare i valori di default. Scorri in basso e fai clic sul pulsante <kbd>Create Repository</kbd>.
+
+4. Ti verranno presentate altre opzioni per i passi successivi, ma non ne avrai bisogno, semplicemente prendi nota dell'URL del tuo repository. A questo  punto puoi uscire dalla pagina senza fare nient'altro.
+
+
+## Salvare (commit) il codice locale su GitHub
+
+Nella sezione precedente hai fatto alcune modifiche al contenuto della pagina. Questo significa che i file del tuo progetto sono cambiati, quindi VS Code mosterà un numero sopa l'icona "Source" nel menu. La scheda "Source" ti sarà utile per fare il commit dei tuoi file periodicamente su GitHub.
+
+1. Fai clic sulla scheda "Source Control" in VS Code per vedere un elenco di file che sono cambiati. Se vedi un messaggio che ti chiede di installare `git`, segui le istruzioni e poi riavvia VS Code.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:43]:
```suggestion
1. Fai clic sulla scheda "Controllo del codice sorgente" in VS Code per vedere un elenco di file che sono cambiati. Se vedi un messaggio che ti chiede di installare `git`, segui le istruzioni e poi riavvia VS Code.
```

See above.

File: src/content/docs/it/tutorial/1-setup/4.mdx (hunk 2 above)
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:47]:
```suggestion
3. Seleziona <kbd>Aggiungi repository remoto da GitHub</kbd>. Se necessario autenticati seguendo le istruzioni, poi ritorna a VS Code e ripeti questa azione.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+
+2. Fai clic sul menu <kbd>•••</kbd> "3 puntini" appena sopra il messaggio di commit e scegli <kbd>Remote</kbd> > <kbd>Add Remote</kbd>.
+
+3. Selezione <kbd>Add remote from GitHub</kbd>. Se necessario autenticati seguento le istruzioni, poi ritorna a VS Code e ripeti questa azione.
+
+4. Dovresti vedere un elenco di tutti i tuoi repository su GitHub. Seleziona quello che hai creato per questo progetto. Se non vedi il tuo progetto, incolla direttamente il suo URL di GitHub. Potrebbe venirti chiesto di dare un nome al repository locale. Puoi scegliere il nome che preferisci. 
+
+5. Nella parte superiore del pannello è possibile inserire un **messaggio di commit** (cioè la descrizione dei cambiamenti ai tuoi file). Scrivi `commit iniziale` e fai clic sul pulsante <kbd>Commit</kbd> per fare il commit delle tue modifiche.
+
+6. Potresti vedere un messaggio che ti informa che non hai "staged commits", e che ti chiede se vuoi inserirli nell'area di stage. Fai clic su <kbd>Always</kbd> e continua.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:53]:
```suggestion
6. Potresti vedere un messaggio che ti informa che non hai "commit preparate", e che ti chiede se vuoi inserirli nell'area di preparazione. Fai clic su <kbd>Sempre</kbd> e continua.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+7. Infine, l'elenco dei file modificato dovrebbe essere sostituito da un pulsante <kbd>Publish</kbd>. Fai clic per inviare i tuoi file salavati su GitHub.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:55]:
```suggestion
7. Infine, l'elenco dei file modificato dovrebbe essere sostituito da un pulsante <kbd>Pubblica</kbd>. Fai clic per inviare i tuoi file salvati su GitHub.
```

File: src/content/docs/it/tutorial/1-setup/4.mdx (hunk 4 above)
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/4.mdx:71]:
```suggestion
- <p>[Utilizzare il sistema di versionamento Git in VS Code](https://code.visualstudio.com/docs/sourcecontrol/overview#_git-support) <Badge>esterno</Badge></p>
```

File: src/content/docs/it/tutorial/1-setup/5.mdx [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+In questa sezione connetterai il tuo repository GitHub con Netlify. Netlify userà il tuo progetto per costruire e pubblicare online il tuo sito sul web ogni volta che farai un nuovo commit al tuo codice.
+
+:::tip[useremo...]
+Questo tutorial usa **Netlify** ma puoi optare per un servizio di hosting a tua scelta per pubblicare il tuo sito su internet. 
+:::
+
+## Crea un nuovo sito Netlify
+
+1. Crea un account gratuito su [Netlify](https://netlify.com) se ancora non ne hai uno. 
+    Prendi nota del tuo username. Potrai vedere il tuo pannello di controllo a tutti i siti che hai creato all'indirizzo `https://app.netlify.com/teams/username`
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/5.mdx:32]:
```suggestion

    Prendi nota del tuo username. Potrai vedere il tuo pannello di controllo a tutti i siti che hai creato all'indirizzo `https://app.netlify.com/teams/username`
```

Missing return.

File: src/content/docs/it/tutorial/1-setup/5.mdx [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+2. Fai clic su <kbd>Add new site</kbd> > <kbd>Import an existing project</kbd>. 
+
+    Ti verrà richiesto di connetterti a un servizio Git. Scegli GitHub e segui i passi che ti vengono mostrati a video per autenticare il tuo accoung GitHub.
+    Poi scegli il repository GitHub del tuo progetto Astro dall'elenco mostrato.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/5.mdx:37]: Here you going to the next line, but you shouldn't.

```suggestion

```

File: src/content/docs/it/tutorial/1-setup/5.mdx (hunk 6 above)
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/5.mdx:36]:
```suggestion
    Ti verrà richiesto di connetterti a un servizio Git. Scegli GitHub e segui i passi che ti vengono mostrati a video per autenticare il tuo account GitHub. Poi scegli il repository GitHub del tuo progetto Astro dall'elenco mostrato.
```

File: src/content/docs/it/tutorial/1-setup/5.mdx [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+
+3. Come passo finale, Netlify ti mostrerà le impostazioni del sito della tua app. I valori predefiniti dovrebbero andare bene per il tuo progetto Astro, quindi scorri in basso e fai clic su <kbd>Deploy site</kbd>.
+
+Complimenti, ora hai un sito Astro!
+
+## Cambia il nome del tuo progetto
+
+Nell'anteprima del tuo sito su Netlify noterai che il tuo progetto avrà un nome generato casualmente, e l'URL del tuo sito sarà nella forma `https://project-name-123456.netlify.app`. Se vuoi puoi cambiare il nome in qualcosa di più semplice da memorizzare, e l'URL si aggiornerà automaticamente di conseguenza.
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/5.mdx:45]:
```suggestion
Nell'anteprima del tuo sito su Netlify noterai che il tuo progetto avrà un nome generato casualmente, e l'URL del tuo sito sarà nella forma `https://nome-progetto-123456.netlify.app`. Se vuoi puoi cambiare il nome in qualcosa di più semplice da memorizzare, e l'URL si aggiornerà automaticamente di conseguenza.
```

It's not a massive change, but let's translate everything.

File: src/content/docs/it/tutorial/1-setup/5.mdx [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+
+<Checklist>
+- [ ] Posso vedere il mio sito online aggiornato.
+- [ ] Sono pronto per ricominciare col codice.
+</Checklist>
+</Box>
+
+### Resources
+
+- <p>[Una guida passo passo per pubblicare su Netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) <Badge>external</Badge></p>
```
Comment (itsmatteomanf) [inline src/content/docs/it/tutorial/1-setup/5.mdx:82]:
```suggestion
- <p>[Una guida passo passo per pubblicare su Netlify](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) <Badge>esterno</Badge></p>
```

**Review by itsmatteomanf — CHANGES_REQUESTED** [review]
Not bad for a first PR! A few changes :)

**Review by lutaok — COMMENTED** [review]
LGTM

**Review by lutaok — APPROVED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 💜

**Pull request merged**